main.go

```diff
@@ -179,13 +199,32 @@ func outputJSON(msg *sarama.ConsumerMessage, messageBytes chan []byte) {
 
 // Validates the flags to make sure all mandatory flags have been supplied
 // throw an error if not the case
-func validateFlags() {
+func validateFlags() error {
+	args := os.Args
+	var error int8 = 0
 	flag.VisitAll(func(f *flag.Flag) {
```

Comment:
This only visits flags that have been passed in on the command line meaning that if you don't pass in a flag it does not get validated.

Test by running the tool with no params, here's the output:

```
$ ./kafka-management-tool
---------------------
Parameters:
broker:
json-out: 0
offset:
partition: 0
schema:
schema-registry:
topic:
---------------------
error creating consumer
panic: kafka: client has run out of available brokers to talk to (Is your cluster reachable?)

goroutine 1 [running]:
main.main()
	main.go:54 +0x379
```

Reply:
this seems to be an issue with the custom `contains` function used in the if statement on the line below

Reply:
can remove the contains func completely and the reference to it (also remove `args = os.Args` above) as it complicates the issue and is not required